lib/board.rb

```diff
@@ -117,4 +117,19 @@ def place(ship, placement_coordinates)
     end
     #may need to add error/return value for non placement
   end
+
+  def render
+    rendered_cells = @cells.map do |coordinate, cell|
+      cell.render
+    end
+    row_arrays = rendered_cells.each_slice(4).to_a
+    rows = row_arrays.map do |array|
+      array.join(' ')
+    end
+
+    range = "A".."D"
+    row_letters = range.to_a
```

Comment:
Maybe the heaviest lift that the render method should be the printing of board/ship placements?

Reply:
Yep, definitely agreed